hydra-node/src/Hydra/Ledger/Cardano/Configuration.hs

```diff
@@ -79,5 +77,5 @@ newLedgerEnv pparams =
       -- both unused in Hydra. There might be room for interesting features in the
       -- future with these two but for now, we'll consider them empty.
       Ledger.ledgerAccount = Ledger.AccountState mempty mempty
-    , Ledger.ledgerPp = toLedgerPParams ShelleyBasedEraAlonzo pparams
+    , Ledger.ledgerPp = toLedgerPParams ShelleyBasedEraBabbage pparams
```

Comment:
We should make sure to also adjust the sample protocol parameters we have to be Babbage's params instead of Alonzo (i.e. remove `d` and `extraEntropy`)

Reply:
Updated the included protocol-parameters.json with ones from the vasil testnet (also has updated cost models).

Reply:
Note that the one retrieved from the vasil testnet using the cardano-cli did include d and entropy as `null` fields:

```
[...]
    "decentralization": null,
    "executionUnitPrices": {
        "priceMemory": 5.77e-2,
        "priceSteps": 7.21e-5
    },
    "extraPraosEntropy": null,
[...]
```

Reply:
Yes, because the cardano-cli / cardano-api have only one data-type to represent both Alonzo and Babbage params, with `Maybe` values :grimacing: ...